.cursor/README.md

```diff
@@ -0,0 +1,62 @@
+# .cursor Directory
+
+This directory contains configuration files for **Cursor IDE**, an AI-powered code editor. These files help AI assistants understand the project context and follow project-specific guidelines when providing code suggestions and assistance.
+
+## Purpose
+
+The `.cursor` directory serves as a way to provide persistent context to AI assistants working in this codebase. When you use Cursor's AI features (Composer, Chat, Autocomplete), the AI can reference these rules to better understand:
+
+- Project architecture and structure
+- Technology stack and dependencies
+- Coding conventions and style guidelines
+- Common patterns and workflows
+- Best practices specific to this project
+
+## Contents
+
+### `rules/project-rule.mdc`
+
+This file contains the **Transition Project - AI Context Guide**, which is automatically applied to all AI interactions in this workspace. It includes:
```

Comment:
I think this is too much details. The description of the file should be in the file itself. 

(I know the AI wrote this, but it went a bit too verbose)